app/controllers/publishers/notifications_controller.rb

```diff
@@ -12,11 +12,11 @@ def index
 
   def notifications
     @notifications ||= current_publisher.notifications
-                                        .created_within_data_access_period
```

Comment:
This scope is on the old notifications model, which will be deleted. It is only referenced here.